src/resolvers/mutation/delete-deployment/index.unit.test.js

```diff
@@ -87,7 +87,7 @@ describe("deleteDeployment", () => {
       );
 
       expect(res.errors).toBeUndefined();
-      expect(deleteDeployment.mock.calls.length).toBe(1);
+      expect(updateDeployment.mock.calls.length).toBe(1);
```

Comment:
It's probably important that we test _what_ we update about the deployment, so all of the cases in this should look something like this:
```suggestion
      expect(updateDeployment).toBeCalledTimes(1);
      expect(updateDeployment).toBeCalledWith({
        where: { id },
        data: { deletedAt: expect.any(Date) }
      });
```

(`toBeCalledTimes` is just my personal preference, but is functionally the same as what is already there)